warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1850,135 +1852,22 @@ private void loadDataTypeMetadata(Multimap<String,Type<?>> fieldToDatatypeMap, S
         }
     }
 
-    protected Set<String> upcase(Set<String> fields) {
```

Comment:
While in general I am a big fan of slimming down the DefaultQueryPlanner, it's hard to see how much of this refactoring is necessary for the validate endpoint.   Is this necessary?

Reply:
The functionality for uppercasing identifiers and applying the query model was extracted to `datawave.query.util.ShardQueryUtils` so that it could be used in `ShardQueryLogic.validateQuery()` when applying some basic normalization on the JEXL query before running it through any JEXL-specific validation. I did this because I didn't want `ShardQueryLogic.validateQuery()` to be too tightly bound DefaultQueryPlanner.

Reply:
Got it!  Thanks!